protocol/streaming/full_node_streaming_manager.go

```diff
@@ -501,8 +503,8 @@ func (sm *FullNodeStreamingManagerImpl) FlushStreamUpdatesWithLock() {
 		}
 	}
 
-	clear(sm.streamUpdateCache)
-	clear(sm.streamUpdateSubscriptionCache)
+	sm.streamUpdateCache = nil
```

Comment:
will this work? vs. setting to empty array

Reply:
yep, tested